src/compiler/transformers/transform-utils.ts

```diff
@@ -483,16 +510,31 @@ const getTypeReferenceLocation = (typeName: string, tsNode: ts.Node): d.Componen
     return true;
   }) as ts.ImportDeclaration;
 
-  if (importTypeDeclaration) {
+  const namedImportBindings = importTypeDeclaration?.importClause?.namedBindings;
+  if (importTypeDeclaration && ts.isNamedImports(namedImportBindings)) {
```

Comment:
For my own edification, why do we need to expand this conditional?

Reply:
it's just a typing hack, basically the `namedBindings` prop could have a few different types so the `ts.isNamedImports` is to narrow it so that `.elements` will be typed correctly